StratisCore.UI/src/app/shared/services/wallet.service.ts

```diff
@@ -105,9 +105,9 @@ export class WalletService extends RestApi {
     return this.getWalletHistorySubject(this.currentWallet).asObservable();
   }
 
-  public estimateFee(feeEstimation: FeeEstimation): Observable<any> {
+  public estimateFee(feeEstimation: FeeEstimation, toSidechain: boolean): Observable<any> {
```

Comment:
For consistency with `accountsEnabled`, could we remove the `toSidechain` param and replace it with `this.globalService.getSidechainEnabled()`?